src/classes/CMT_MetadataAPI.cls

```diff
@@ -170,68 +192,29 @@ public class CMT_MetadataAPI {
             Map<String, String> statusMap = new Map<String, String>{ this.jobId => (result != null ? result.status.name() : 'Unknown') };
-
             // If the status is not Success, collect all errors into a string that can be written as the status
             if (result != null && result.status != Metadata.DeployStatus.Succeeded) {
-                setStatusMap(result, statusMap, isSuccess);
-            }
-
-            if (source == CMT_MetadataAPI.Source.HELP_TEXT_MIGRATION) {
-                setHelpTextMigrationSettings(statusMap, isSuccess);
-            } else {
-                setCustomizableRollupsSettings(statusMap, isSuccess);
-            }
-
-            if (isSuccess) {
-                UTIL_OrgTelemetry_SVC.asyncProcessNpspSettingsTelemetry();
-            }
-        }
-
-        /**
-        * @description Method that parses the Metadata deployment result into a map, job id => status
-        */
-        private void setStatusMap(Metadata.DeployResult result, Map<String, String> statusMap, Boolean isSuccess) {
-            String errMsg = '';
-            for (Integer n=0; n < result.details.componentFailures.size(); n++) {
-                errMsg += (n > 0 ? '\n' : '') + '* ' + result.details.componentFailures[n].fullName +
-                        '> ' + result.details.componentFailures[n].problem;
+                String errMsg = '';
+                for (Integer n=0; n < result.details.componentFailures.size(); n++) {
+                    errMsg += (n > 0 ? '\n' : '') + '* ' + result.details.componentFailures[n].fullName +
+                            '> ' + result.details.componentFailures[n].problem;
+                }
+                // The custom settings field is a max of 255 characters, but need space for the serialized json overhead
+                statusMap.put(this.jobId, errMsg.left(220));
+                isSuccess = false;
             }
 
-            // The custom settings field is a max of 255 characters, but need space for the serialized json overhead
-            statusMap.put(this.jobId, errMsg.left(220));
-            isSuccess = false;
-        }
-
-        /**
-        * @description Update Customizable Rollups Settings
-        */
-        private void setCustomizableRollupsSettings(Map<String, String> statusMap, Boolean isSuccess) {
             // Write the deployment status to the custom settings object
             Customizable_Rollup_Settings__c crlpSettings = UTIL_CustomSettingsFacade.getCustomizableRollupSettings();
             crlpSettings.CMT_API_Status__c = JSON.serializePretty(statusMap).left(255);
-
+            // THIS WILL NEED TO BE REFACTORED IF THIS CMT DEPLOYMENT CODE IS USED BY SOMETHING OTHER THAN ROLLUPS
             crlpSettings.Customizable_Rollups_Enabled__c = (isSuccess || crlpSettings.Customizable_Rollups_Enabled__c == true);
             if (!Test.isRunningTest()) {
                 upsert crlpSettings;
             }
+            if (isSuccess) {
```

Comment:
Is this change meant to be included in this PR?

Reply:
Hmm I'm pretty sure this mirrors what we have in master currently. Does git think this is a new addition because I have previous commits that moved things around in here?

Reply:
Possibly - it looks unchanged to me, other than being pushed down in the file because of your additions above this line.  But strange that it is included in your PR.  Maybe try checking the file out from Master and then re-applying your changes to this file on top and see if this goes away?